Dockerfile

```diff
@@ -19,5 +19,12 @@ FROM debian:bullseye-slim as runner
 
 WORKDIR /app
 
+RUN apt-get update && \
+    apt-get install -y git && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
For my understanding, why do we need these last two commands?

Reply:
it removes unused files, so that the image takes less space.
It's something we also do at line 11.

I found an explanation [here](https://stackoverflow.com/questions/61990329/benefits-of-repeated-apt-cache-cleans)